docs/doxygen.cfg

```diff
@@ -0,0 +1,42 @@
+# This is a slightly modified standard doxygen config file
+
+DOXYFILE_ENCODING      = UTF-8
+PROJECT_NAME           = "Ert"
+PROJECT_NUMBER         = 2.1
```

Comment:
This should be created with the cmake `configure_file( )` functionality, getting the version numbers from the cmake variables [here](https://github.com/Ensembles/ert/blob/master/CMakeLists.txt#L10).